Uses private parts of the QtQuick3D api which added a namespace in:
https://github.com/qt/qtquick3d/commit/2b139225745a94ab4fa7e30f28cd0207469ee68e
--- a/src/tools/qml2puppet/qml2puppet/editor3d/generalhelper.cpp
+++ b/src/tools/qml2puppet/qml2puppet/editor3d/generalhelper.cpp
@@ -803,5 +803,9 @@
     const QMatrix4x4 sceneTransform = parent->sceneTransform() * localTransform;
 
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 5, 3))
+    return QSSGUtils::mat44::getPosition(sceneTransform);
+#else
     return mat44::getPosition(sceneTransform);
+#endif
 }
 
--- a/src/tools/qml2puppet/qml2puppet/editor3d/mousearea3d.cpp
+++ b/src/tools/qml2puppet/qml2puppet/editor3d/mousearea3d.cpp
@@ -738,5 +738,9 @@
     const QMatrix4x4 sceneTransform = parent->sceneTransform() * localTransform;
 
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 5, 3))
+    return QSSGUtils::mat44::getPosition(sceneTransform);
+#else
     return mat44::getPosition(sceneTransform);
+#endif
 }
 
